components/PollCard.tsx

```diff
@@ -0,0 +1,121 @@
+'use client';
+
+'use client';
```

Comment:
You only need to specify use client once :)

Reply:
I know :)